src/poliastro/twobody/orbit.py

```diff
@@ -1094,9 +1095,13 @@ def plot(self, label=None, use_3d=False):
             Label for the orbit, defaults to empty.
         use_3d : bool, optional
             Produce a 3D plot, default to False.
-
+        static : bool, optional
+            Produce a static image of the figure, default to false
         """
-        if use_3d:
+        from poliastro.plotting.static import StaticOrbitPlotter
+
+        if static:
+            return StaticOrbitPlotter().plot(self, label=label)
+        elif use_3d:
```

Comment:
Use just a if block! 
```suggestion
        if static:
            return StaticOrbitPlotter().plot(self, label=label)
        if use_3d:
            return OrbitPlotter3D().plot(self, label=label)
        else:
            return OrbitPlotter2D().plot(self, label=label)    
         
```
In that way you won't have to write down extra tests!

Reply:
Also, the Pull Request can't be merged until you get all the ticks green in the checks :)

Reply:
I did use the if block, but the codecov tests still fail. [https://codecov.io/gh/poliastro/poliastro/compare/d83b933155b6f5d97c1025cf953247da8d11bbad...91f1e933b88214bae33b6710833a49f9644f88f4/diff](The report)

Reply:
I did add an if block but the codecov tests still fail [The report](https://codecov.io/gh/poliastro/poliastro/compare/d83b933155b6f5d97c1025cf953247da8d11bbad...91f1e933b88214bae33b6710833a49f9644f88f4/diff)